src/services/videoconference/index.js

```diff
@@ -0,0 +1,135 @@
+const {
+	BadRequest,
+	Forbidden,
+	NotFound,
+	GeneralError,
+	NotImplemented,
+	FeathersError,
+} = require('@feathersjs/errors');
+const { FEATURE_VIDEOCONFERENCE_ENABLED } = require('../../../config/globals');
+
+const { error } = require('../../logger');
+const videoconferenceHooks = require('./hooks');
+
+const { ScopePermissionService } = require('../helpers/scopePermissions');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['ScopePermissionService' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)